.github/workflows/close-incomplete-issues.yml

```diff
@@ -11,8 +11,8 @@ jobs:
   close-issues-if-invalid:
     runs-on: ubuntu-latest
     steps:
-      - uses: queengooborg/invalid-issue-closer@43bf1ddff3e83d208b77a3e364f7b5e22f6a9657 # v1.6.0
-        id: spam-check
+      - uses: queengooborg/invalid-issue-closer@v1.6.0
```

Comment:
For security reasons, please revert to pin to the hash. (Dependabot can handle this.)

Reply:
Question: what is the security benefit of using the specific hash in this case?

Reply:
Pinning GitHub actions to a commit protects against supply chain attacks. If a malicious attacker gets write access to the action repo, they might be able to change the tag to point to a different commit, without anyone noticing it.

So for security reasons, and consistency, we now require GitHub actions to be pinned, except for our own actions (mdn/workflows), and official GitHub actions (actions/*).